src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -106,9 +106,12 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
 
   @volatile private var sessionUsed = false
 
+  private val metricsCategory = s"cassandra.readjournal.${system.name}.hash.${System.identityHashCode(this)}"
```

Comment:
I wouldn't mind working on that but I was thinking for now I can make it like:
`s"cassandra-query-journal.${System.identityHashCode(this)}"`

With a TODO comment that say it should change as soon as akka#19822 is fixed. Is that a good idea?